modules/ROOT/pages/about-api-catalog-cli.adoc

```diff
@@ -0,0 +1,689 @@
+. Select *Users*.
+. Select the user for which you want to enable the permission. 
+. In the *Permissions* tab, select the *API Catalog* subtab.
+. Select *API Catalog Contributor*. 
+
+== Create a Descriptor File
+
+Before you run the commands to publish your APIs, you must create a descriptor file that has the information Exchange needs to catalog the APIs.  
+
+=== Create a Descriptor File Using the CLI
+
+You can run the `api-catalog create-descriptor` command to create and print the catalog descriptor YAML results to a designated file. The command will output cataloging information for all API specifications it finds in the full directory tree relative to the current working directory.
+
+To create a descriptor file using the CLI:
+
+. Navigate to the directory path that contains the APIs you want to catalog. 
+. Run the `api-catalog create-descriptor` command. For example:
++
+----
+> api-catalog create-descriptor -f mydescriptor.yaml
+----
+
+The result of the command depends on the presence and content of the designated descriptor file as follows:
+
+* No file or empty file: The command will discover the APIs and write the results to standard output and to a file named `catalog.yaml`.
+
+* Existing file with content: The command will fail and show an error message that says the descriptor file already exists.
+
+NOTE: You can optionally create a descriptor file manually instead of using the `api-catalog create-descriptor` command to generate the file. See <<Create a Descriptor File Manually>>.
+
+//Limitations:
+//* Each API specification must have a single API.  
+//* You cannot log in through the CLI if you system uses multi-factor authentication. Your system must use SSO and you must log in using the client ID and client secret of the Connected App.
+
+==== api-catalog create-descriptor
+
+Following is a reference for the `api-catalog create-descriptor` command. 
+
+*Usage*
+
+----
+> api-catalog create-descriptor [options] 
+----
+
+*Options*
+
+`-f, --file=file`
+  +
+  [default: `catalog.yaml`] 
+  + 
+  The name and location in which to save the generated catalog descriptor file.
+
+== Publish APIs Using CLI
+
+You can use the `api-catalog publish-asset` command to publish your APIs to Exchange. You will typically embed this command in your automation tools, such as a CI/CD pipeline or custom scripts, to automatically trigger the cataloging of your API assets. The command publishes your APIs using the information specified in the `--descriptor-file` option.   
+
+=== Prerequisites
+
+* Create a descriptor file and copy it to the file path where your API projects are stored. See <<Create a Descriptor File>>.
+
+To publish an API using the CLI:
+
+* Run the `api-catalog publish-asset` command. For example:
++
+----
+> api-catalog publish-asset -a https://anypoint.mulesoft.com -d catalog.yaml -o 1234abc-b34a-7cd3-4s32-12345abc2345 -u myAnyPtAccount -p myPwd@4!myacct 
+----
+
+=== api-catalog publish-asset
```

Comment:
This is unique among the main 4 tasks in that it has just one prereq and just one step before the reference.
Let's chat about this.

Reply:
Yes, I have several examples after the reference section, and there really isn't much to say in the steps other than to run the command. I could definitely integrate the prereq as step 1.

Reply:
Actually, for this one I want to keep the prerequisite separate. Creating the descriptor file is not part of the publishing procedure, and this could confuse the readers.